examples/send_from_file/example.exs

```diff
@@ -161,7 +161,7 @@ defmodule Peer do
         Process.send_after(self(), :send_audio_packet, duration)
         # values set to 0 are handled by PeerConnection.set_rtp
         rtp_packet = OpusPayloader.payload(packet)
-        rtp_packet = %{rtp_packet | timestamp: state.last_audio_timestamp}
+        rtp_packet = %{rtp_packet | timestamp: trunc(state.last_audio_timestamp)}
```

Comment:
Can this ever be non-integer? We calculate last_audio_timestamp as:  `last_audio_timestamp: state.last_audio_timestamp + timestamp_delta` where timestamp_delta is `timestamp_delta = trunc(duration * 48_000 / 1000)` (so integer)  and initial last_audio_timestamp is also integer

Reply:
Yeas, this is just temporary (duration from `OggReader.next_packet` is now a float), I'll make there's no unnecessary truncs in the examples PR.